lib/metadata_json_lint.rb

```diff
@@ -136,10 +142,26 @@ def validate_requirements!(requirements)
       if requirement['name'] == 'pe'
         warn :requirements, "The 'pe' requirement is no longer supported by the Forge."
       end
+
+      begin
+        VersionRequirement.new(requirement.fetch('version_requirement', ''))
+      rescue ArgumentError => e
+        # Raised when the version_requirement provided could not be parsed
+        error :dependencies, "Invalid 'version_requirement' field in metadata.json: #{e}"
+      end
+
+      validate_puppet_ver!(requirement) if options[:strict_puppet_version]
```

Comment:
Here you can use the parsed `VersionRequirement` rather than the raw requirement. If it failed to parse, validating the version is likely to fail and I wouldn't bother validating the content.